src/client/theme-default/components/VPDocFooterLastUpdated.vue

```diff
@@ -13,9 +13,27 @@ const datetime = ref('')
 // set time on mounted hook to avoid hydration mismatch due to
 // potential differences in timezones of the server and clients
 onMounted(() => {
+  function findBestLocaleMatch(pageLocale: string): string | undefined {
+    return navigator.languages.find((userLang) => {
+      if (pageLocale === userLang)
+        return pageLocale
```

Comment:
these should be just return true? (find expects boolean function) 👀

Reply:
arggg